pkg/lib/routines/routines.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package routines
+
+import (
+	"fmt"
+
+	"github.com/cortexlabs/cortex/pkg/lib/exit"
+)
+
+func RunWithPanicHandler(f func()) {
+	go func() {
+		defer func() {
+			if r := recover(); r != nil {
+				err := ErrorUnexpectedError(fmt.Sprintf("%v", r))
```

Comment:
I think this error can be removed like the other one, and instead we can use `err := errors.CastRecoverError(r)`